myapp/app/views/tasks/show.html.erb

```diff
@@ -1,11 +1,11 @@
-<h1>Task Detail</h1>
+<h1><%= t '.title' %></h1>
 
-<h2>Title: <%= @task.title %></h2>
-<p>Description: <%= @task.description %></p>
+<h2><%= @task.title %></h2>
+<p><%= (Task.human_attribute_name :description) + ":" +  @task.description %></p>
```

Comment:
I think it is better to use "string interpolation" when concatenating characters in ruby 👍 
https://linuxhint.com/ruby-string-interpolation/

```suggestion
<p><%= "#{Task.human_attribute_name :description}:#{@task.description}" %></p>
```

Reply:
Thank you for telling me.
I fixed it